content/docs/guides/components/modeldb.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "ModelDB"
+description ="ModelDB - A system to manage machine learning models"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+
+## Using ModelDB
+
+After ModelDB is deployed using kfctl.sh and the modeldb-db, modeldb-frontend and modeldb-backend pods are running - 
```

Comment:
ModelDB isn't installed by default. People will need to install it by doing
ks generate ..
ks apply ...

Can you add the appropriate instructions please?

Reply:
FYI @garganubhav this changed in the latest PR. So users now have to run ks commands manually to setup.